SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  ViewController.swift
+//  SupportClient
+//
+//  Created by Omar Rasheed on 2/15/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    private let feedbackLabel = UILabel()
+    private let feedbackTableView = UITableView(frame: .zero)
+    private let newConversationButton = UIButton()
+    private let searchController = UISearchController() // TODO: For later
+        
+    var feedbackData = [Feedback]()
+    var filteredFeedbackData = [Feedback]() // TODO: For later
+    
+    private(set) var countEditTaps: Int = 0
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.backgroundColor
+        // Do any additional setup after loading the view.
+        
+        setupData()
+        setupNavigationBar()
+        setupFeedbackLabel()
+        setupConversationButton()
+        setupFeedbackTableView()
+        setupConstraints()
+    }
+    
+    // TODO: hook up to actual server to load the data
+    // This is dummy data for testing
+    // Add variables to feedbackData to show the tableView + editing functionality
+    func setupData() {
+                
+        let jsonString = """
+        {
+            "title" : "Ithaca Transit Bug",
+            "message" : "This app sometimes glitches out on me and shows the wrong bus times",
+            "hasRead" : false
+        }
+        """
+        
+        let jsonData = jsonString.data(using: .utf8)!
+        let feedback = try! JSONDecoder().decode(Feedback.self, from: jsonData)
```

Comment:
i know its just placeholder but just for the future, we should avoid using `!` and unwrap using `if-let` or `guard-let`